TreeTracker/UI/Views/Home/HomeViewController.swift

```diff
@@ -84,6 +84,12 @@ class HomeViewController: UIViewController, AlertPresenting {
     override func viewDidLoad() {
         super.viewDidLoad()
         viewModel?.fetchTrees()
+        self.showLogoutButton()
+    }
+
+    private func showLogoutButton() {
+        let logoutButton = UIBarButtonItem(title: "Log Out", style: .plain, target: self, action: #selector(logoutButtonPressed))
```

Comment:
Would be good to add the "Log Out" text to Localizable.strings then use the SwiftGen generated `L10n` value here.